cardano-submit-api/testing-tx-submit.md

```diff
@@ -0,0 +1,73 @@
+# Testing cardano-submit-api
+
+Setting this up for testing and for actual use on a real network is significantly more difficult
+than it should be. This is definitely not ready for end users yet.
+
+The biggest thing missing at the moment is documentation of generation of transactions with the
+command line tools. That needs to be *thoroughly* documented and this document is not the place
+for that.
+
+The following instructions are for generating a testnet and testing tx-submission on that testnet.
+
+### Start a local shelley testnet
+
+*   Run the testnet using the following script:
+
+    ```bash
+    cardano-node $ ./scripts/lite/shelley-testnet.sh
+    ```
+
+*   Generate a transaction:
+
+    ```bash
+    cardano-node $ ./scripts/benchmarking/issue-genesis-utxo-expenditure.sh genesis_transaction.bin
```

Comment:
Currently getting: 
```
./scripts/benchmarking/issue-genesis-utxo-expenditure.sh genesis_transaction.bin
Invalid option `--config'
```
I think it would probably be better to create a folder in the `scripts` folder to house a script that creates a transaction. The reason being is that I don't maintain any of the scripts in the `benchmarking` folder (they are complex)